maven-surefire-plugin/src/site/apt/examples/toolchains.apt.vm

```diff
@@ -0,0 +1,56 @@
+ ------
+ Using Maven Toolchains
+ ------
+ Akom <akom>
+ ------
+ 2020-04-17
+ ------
+
+~~ Licensed to the Apache Software Foundation (ASF) under one
+~~ or more contributor license agreements.  See the NOTICE file
+~~ distributed with this work for additional information
+~~ regarding copyright ownership.  The ASF licenses this file
+~~ to you under the Apache License, Version 2.0 (the
+~~ "License"); you may not use this file except in compliance
+~~ with the License.  You may obtain a copy of the License at
+~~
+~~   http://www.apache.org/licenses/LICENSE-2.0
+~~
+~~ Unless required by applicable law or agreed to in writing,
+~~ software distributed under the License is distributed on an
+~~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+~~ KIND, either express or implied.  See the License for the
+~~ specific language governing permissions and limitations
+~~ under the License.
+
+~~ NOTE: For help with the syntax of this file, see:
+~~ http://maven.apache.org/doxia/references/apt-format.html
+
+    For general information about Maven Toolchains, see
+    {{{https://maven.apache.org/guides/mini/guide-using-toolchains.html}Guide to Using Toolchains}}
+
+
+Using Maven Toolchains with Surefire/Failsafe.
+
+    By default, if the pom configures the toolchains plugin as specified in the aforementioned
+    guide, surefire and failsafe will launch the test jvm using the main toolchain
```

Comment:
Pls substitute `surefire and failsafe` with the expression `${thisPlugin}`, thx.